infrastructure/testdriver/bidi/bluetooth/simulate_adapter.https.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<meta charset="utf-8"/>
+<title>TestDriver bidi.permissions.set_permission method</title>
```

Comment:
Copy & paste mistake? This is a test for bidi.bluetooth.simulate_adapter, right?

Reply:
Thanks, it does look like a copy-paste mistake so I went ahead an applied the fix. @sadym-chromium I also rebased so the PR so that it could be landed.

Reply:
Nice catch and thanks for the fix!